tests-ui/tests/composables/graph/useSelectionState.test.ts

```diff
@@ -267,4 +268,57 @@ describe('useSelectionState', () => {
       expect(newIsPinned).toBe(false)
     })
   })
+
+  describe('Help Sync', () => {
+    test('opens help for newly selected node when help is open', async () => {
+      const nodeDefStore = useNodeDefStore() as any
+      nodeDefStore.fromLGraphNode.mockImplementation((node: TestNode) => ({
+        nodePath: node.type
+      }))
+
+      const nodeHelpStore = useNodeHelpStore() as any
+      nodeHelpStore.isHelpOpen = true
+      nodeHelpStore.currentHelpNode = { nodePath: 'NodeA' }
+
+      const nodeA = createTestNode({ type: 'NodeA' })
+      mockSelectedItems.value = [nodeA]
+
+      useSelectionState()
+
+      const nodeB = createTestNode({ type: 'NodeB' })
+      mockSelectedItems.value = [nodeB]
+
+      await nextTick()
```

Comment:
Could you use some of the test helpers to interact with this more like a user would? That way you wouldn't have to manually manage the DOM updates.
https://test-utils.vuejs.org/guide/advanced/async-suspense